Fixes the following error:

    In file included from alias.c:15:0:
    file.h:9:47: error: unknown type name 'mode_t'
     int ensure_directories_exist(const char *str, mode_t dir_mode);
                                                   ^~~~~~
    make[1]: *** [../Makefile.inc:134: alias.o] Error 1

--- a/libmultipath/alias.c
+++ b/libmultipath/alias.c
@@ -8,6 +8,7 @@
 #include <string.h>
 #include <limits.h>
 #include <stdio.h>
+#include <sys/stat.h>
 
 #include "debug.h"
 #include "uxsock.h"
